calc/index.html

```diff
@@ -39,11 +39,23 @@
     var USERMEM = (RAM + SWAP - ((7.5 * 1024) + (RAM * 0.05)));
     var KERNELMEM = (0.026 * (USERMEM / 1.7));
     var OvercommitRatio = Math.floor((((RAM - KERNELMEM) / RAM) * 100) / 5) * 5;
-    var SuggestedVmem = Math.ceil(((((USERMEM - GPDBOtherMem) /1.7) / NumOfSegs) / 1024) * 10) / 10;
+    var SuggestedVmem = Math.round( Math.ceil(((((USERMEM - GPDBOtherMem) /1.7) / NumOfSegs))));
+    
+    console.log(SuggestedVmem)
 
-   
-    $('#vmem-protect-value').html("<h4>gp_vmem_protect_limit = " + SuggestedVmem + "</h4>");
-    $('#vm-overcommit-ratio-value').html("<h4>vm.overcommit_ratio = " + OvercommitRatio + "</h4>");  
+    // convert SuggestedVmem to MB and ensure value is divisible by 100mb for readability
+    SuggestedVmem =  SuggestedVmem - (SuggestedVmem % 100);
```

Comment:
Should we round `SuggestedVmem` to the nearest hundred in either direction rather than always down? Something along these lines will round up or down to the nearest hundred or fallback to `SuggestedVmem` in case it's below 50Mb:
```Javascript
SuggestedVmem = ((Math.floor((SuggestedVmem + 50) / 100) * 100) || SuggestedVmem);
```